jta-1_2-in-wildfly/pom.xml

```diff
@@ -38,21 +38,20 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.target>1.7</maven.compiler.target>
-        <maven.compiler.source>1.7</maven.compiler.source>
         <version.org.jboss.spec.javax.transaction>1.0.1.Final</version.org.jboss.spec.javax.transaction>
         <version.org.jboss.spec.javax.ejb>1.0.0.Final</version.org.jboss.spec.javax.ejb>
         <version.org.jboss.spec.javax.jms>1.0.0.Final</version.org.jboss.spec.javax.jms>
         <version.hibernate.javax.persistence>1.0.0.Final</version.hibernate.javax.persistence>
         <version.javax.enterprise>1.1</version.javax.enterprise>
         <version.junit>4.13.1</version.junit>
-        <version.org.wildfly.arquillian>2.1.1.Final</version.org.wildfly.arquillian>
+        <version.org.wildfly.arquillian>3.0.1.Final</version.org.wildfly.arquillian>
         <version.org.jboss.arquillian>1.1.13.Final</version.org.jboss.arquillian>
         <ipv6.server.jvm.args>-Djboss.bind.address=[::1] -Djboss.bind.address.management=[::1] -Djboss.bind.address.unsecure=[::1] -Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true</ipv6.server.jvm.args>
         <jvm.args.memory>-Xms64m -Xmx1024m -XX:MaxMetaspaceSize=512m</jvm.args.memory>
         <jvm.args.debug></jvm.args.debug>
         <jvm.args.other>-server</jvm.args.other>
         <version.surefire.plugin>2.19.1</version.surefire.plugin>
+        <modular.jdk.args>--add-opens=java.base/java.util=ALL-UNNAMED</modular.jdk.args>
```

Comment:
This is related to --add-opens but in a different way. Given this is the quickstarts project that people can try to understand why things are done in certain ways, maybe we should try to justify a bit more why --add-opens is being used and whether it's just for testing purposes? WDYT?

Reply:
Thanks @tomjenkinson for your comment. Yes, this --add-opens is needed because the wildfly version we are using needs that argument to work in our quickstart. I will add a comment about it.

Reply:
Thanks, I think a comment would be added for each use in each quickstart. If you can share why WildFly needs it it could be good I think.

Reply:
I added the comments @tomjenkinson and you can follow the topic about this here: https://wildfly.zulipchat.com/#narrow/stream/174184-wildfly-developers/topic/InaccessibleObjectException.20.28add-opens.29